README.md

```diff
@@ -363,6 +387,20 @@ typed. It’s not needed to type something when TypeScript can resolve the type.
 Keep your code as strict as possible, so keep all functions and properties `private` unless they
 have to be `protected` or `public`.
 
+Explicitly define whether your `constructor` functions are for internal or external interfaces.
+
+```ts
+// bad
+constructor(...args: Array<any>) {
+  super(...args);
+}
+
+// good
+public constructor(...args: Array<any>) {
+  super(...args);
+}
```

Comment:
This example shows an empty constructor and an `any` type which are both not allowed in the eslint-config. Maybe change this to:

```suggestion
```ts
// bad
constructor() {
  // ...
}

// good
public constructor() {
  // ...
}
```